src/config/dynamic-theme-fixes.config

```diff
@@ -24307,6 +24307,16 @@ IGNORE IMAGE ANALYSIS
 
 ================================
 
+unsplash.com
+
+INVERT
+svg[aria-labelledby="unsplash-home"]
+
+CSS
+li > a[aria-current="page"] {box-shadow: rgb(230,230,230) 0px -2px inset;}
```

Comment:
As we can use dark/light scheme in Dark Reader setting almost white color for that "underline" breaks it in light sheme.
To avoid this we use ${source_color}.

So change `rgb(230,230,230)` to `${rgb(25,25,25)}` to have the same effect in dark scheme but make it dark in light scheme. 😉

Reply:
oh, ok. Didn't know that